frontend/src/components/uam/users-table/styles.module.scss

```diff
@@ -0,0 +1,25 @@
+.usersTable {
```

Comment:
this should be table default styles

Reply:
Does default table have a header in the left upper corner and a shadow block (thisngs I did in users-table). I thought original table's task was only to display info, everything else will be build in other components wrapping default table